cocos/renderer/gfx-gles2/GLES2Commands.cc

```diff
@@ -285,10 +285,11 @@ GLenum GFXFormatToGLType(GFXFormat format) {
         case GFXFormat::RGB9E5: return GL_FLOAT;
 
         case GFXFormat::D16: return GL_UNSIGNED_SHORT;
-        case GFXFormat::D16S8: return GL_UNSIGNED_SHORT;
+        case GFXFormat::D16S8: return GL_UNSIGNED_INT_24_8_OES; // no D16S8 support
         case GFXFormat::D24: return GL_UNSIGNED_INT;
         case GFXFormat::D24S8: return GL_UNSIGNED_INT_24_8_OES;
         case GFXFormat::D32F: return GL_FLOAT;
+        case GFXFormat::D32F_S8: return GL_UNSIGNED_INT_24_8_OES; // no D32FS8 support
```

Comment:
it can be query by :
https://github.com/cocos-creator/cocos2d-x-lite/blob/81dc02a68be3d83e43c29487b19ba8126395c817/cocos/renderer/gfx-gles2/GLES2Device.cc#L105

Reply:
depth_buffer_float 是桌面 gl 的 extension，看起来 es 2 并没有相关支持